apollo-client/src/test/java/com/ctrip/framework/apollo/spring/BootstrapConfigTest.java

```diff
@@ -420,10 +421,10 @@ public static void afterClass() {
 
     @Test
     public void test() {
-      List<EnvironmentPostProcessor> processorList =  SpringFactoriesLoader.loadFactories(EnvironmentPostProcessor.class, getClass().getClassLoader());
+      List<String> names = SpringFactoriesLoader.loadFactoryNames(EnvironmentPostProcessor.class, getClass().getClassLoader());
```

Comment:
What's the difference between `SpringFactoriesLoader.loadFactoryNames` and `SpringFactoriesLoader.loadFactories`?

Reply:
loadFactories need to consturct the factory. It will fail because of `ConfigDataEnvironmentPostProcessor` doesn't have a empty consturctor